dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/pipeline/inbound/HttpDispatcherHandler.java

```diff
@@ -119,7 +119,7 @@ public void exceptionCaught(ChannelHandlerContext context, Throwable cause) thro
             //Legacy doesnt throw ffdc on processNewInformation
             if(context.channel().attr(NettyHttpConstants.THROW_FFDC).get() != null){
                 context.channel().attr(NettyHttpConstants.THROW_FFDC).set(null);
-            }else{
+            }else if(!cause.getMessage().contains("possibly HTTP/0.9")){
```

Comment:
We'll want to check if we should filter all of these HTTP/0.9 or not but this looks okay for now